inc/targetbase.class.php

```diff
@@ -1038,6 +1038,94 @@ protected function parseTags($content, PluginFormcreatorFormAnswer $formanswer,
       return $content;
    }
 
+   protected function showConditionSettings($rand) {
+      global $DB;
+
+      echo '<tr class="line0">';
+      
+      echo '<td width="15%">' . __('Condition') . '</td>';
+      echo '<td width="45%">';
+
+      $options[0] = __('No');
+      $options[1] = __('Yes');
+
+      $params['value'] = $this->fields['has_condition'];
+      $params['on_change'] = 'change_condition()';
```

Comment:
the version 2.9.0 is on its way and I renamed a lot of JS fucntions to add "plugin_formcreator_" prefix. If change_condition is a new functoin, please rename it to **plugin_formcreator_changeCondition()**. Maybe plugin_formcreator_changeDestinationContition()** should be better as there are conditions for questions, and I might implement soon conditions on sections also.

I plan to see how easy your contribution is easy to rebase onto the current 2.9.0 code. A new feature like this PR must be in a new minor version. 

The 2.9 branch is not published yet, sorry for the inconvenience.